docs/evaluate/development-production-features/high-availability.mdx

```diff
@@ -19,111 +19,18 @@ keywords:
   - temporal-cloud
   - term
 ---
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
 
-:::tip Support, stability, and dependency info
+Temporal offers high availability as a feature for both Temporal Cloud and self-hosted deployments. These features ensures your applications remain operational even in the face of regional outages or service disruptions.
 
-- Multi-region Namespaces are in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for Temporal Cloud.
+- [Multi-region Namespaces for Temporal Cloud](/evaluate/development-production-features/multi-region-namespace): Temporal Cloud's Multi-region Namespaces offer automated failover, synchronized data replication, and high availability for workloads requiring disaster-tolerant deployment and 99.99% uptime.
+- [Global Namespace for Self-Hosted](/evaluate/development-production-features/global-namespace): For self-hosted Temporal deployments, Global Namespace provides enhanced availability across multiple Temporal Services, with automatic replication of metadata and Workflow Execution state.
```

Comment:
Still having this on the multi-region namespace page is obviously a bit weird... but I trust that you've given this some thought and if there isn't a better solution, I don't feel strongly about it.